sklearn/linear_model/tests/test_ridge.py

```diff
@@ -692,6 +697,48 @@ def test_sparse_design_with_sample_weights():
                                       decimal=6)
 
 
+testdata_alpha = [
+    ((1, 10, 100), np.array([1.0, 10.0, 100.0])),
+    ((-1, -10, -100), np.array([-1.0, -10.0, -100.0]))
+]
+
+
+@pytest.mark.parametrize("alpha_input, alpha_expected", testdata_alpha)
+def test_conversion(alpha_input, alpha_expected):
+    assert((RidgeCV(alpha_input).get_params()['alphas'] ==
```

Comment:
I'm not actually sure what this is trying to test. Is it trying to test that the input is validated and turned into floats before fit? We don't usually do this, because the user may also set them with set_params.

I also don't think this is currently asserting that the alphas are floats, only that they are unchanged or equivalent.

And I think we have common tests which do that. I'm short, I don't think this test adds anything in its current form.

Reply:
ok, I'll remove it, just put it as a suggestion from the other reviewer. At least I've learned how these tests work.